src/core/lib/security/credentials/external/external_account_credentials.cc

```diff
@@ -55,14 +55,18 @@
 #include "src/core/lib/security/credentials/external/url_external_account_credentials.h"
 #include "src/core/lib/security/util/json_util.h"
 #include "src/core/lib/slice/b64.h"
-#include "src/core/lib/uri/uri_parser.h"
 
 #define EXTERNAL_ACCOUNT_CREDENTIALS_GRANT_TYPE \
   "urn:ietf:params:oauth:grant-type:token-exchange"
 #define EXTERNAL_ACCOUNT_CREDENTIALS_REQUESTED_TOKEN_TYPE \
   "urn:ietf:params:oauth:token-type:access_token"
 #define GOOGLE_CLOUD_PLATFORM_DEFAULT_SCOPE \
   "https://www.googleapis.com/auth/cloud-platform"
+#define IMPERSONATED_CRED_DEFAULT_LIFETIME_IN_SECONDS \
+  1 * 60 * 60                                              // 1 hour - 3600s
+#define IMPERSONATED_CRED_MIN_LIFETIME_IN_SECONDS 10 * 60  // 10 mins - 600s
+#define IMPERSONATED_CRED_MAX_LIFETIME_IN_SECONDS \
+  12 * 60 * 60  // 12 hours - 43200s
```

Comment:
Personally I think calculated value make more sense here if we already have comments following that.